tests/flow.html

```diff
@@ -5,7 +5,7 @@
   <title>VexFlow 2 - JavaScript Music Notation and Guitar Tab</title>
   <link rel="stylesheet" href="flow.css" type="text/css" media="screen" />
   <!-- VexFlow Sources -->
-  <script src="../build/vexflow-debug.js"></script>
+  <script src="../releases/vexflow-debug.js"></script>
```

Comment:
Running `grunt` will put files in `build/`. Also, `grunt watch` will keep `build/` updated as you change sources. The `releases/` directory is what I populate when I cut a new release and publish and NPM.